readthedocs/api/v3/serializers.py

```diff
@@ -88,6 +88,31 @@ def get_project(self, obj):
         return self._absolute_url(path)
 
 
+class BuildURLsSerializer(BaseLinksSerializer, serializers.Serializer):
+    _self = serializers.URLField(source='get_full_url')
```

Comment:
we are not using self in the rest of url serializers, I think we can leave it as `build`

Reply:
sure!